jmxtrans-output/jmxtrans-output-influxdb/src/main/java/com/googlecode/jmxtrans/model/output/InfluxDbWriter.java

```diff
@@ -152,4 +181,12 @@ public void doWrite(Server server, Query query, ImmutableList<Result> results) t
 		}
 		return resultTagMap;
 	}
-}
\ No newline at end of file
+
+	private String getSource(Server server) {
```

Comment:
That's probably something that should be moved directly on the `Server` class. The same kind of behaviour should be used for most output writers.